docs/guides/stay-up-to-date/index.md

```diff
@@ -30,7 +31,6 @@ import CardGroup from "/src/components/CardGroup"
 <Card title="Update to 2023-06" href="/guides/stay-up-to-date/releases/2023-06" />
 <Card title="Update to 2023-05" href="/guides/stay-up-to-date/releases/2023-05" />
 <Card title="Update to 2023-04" href="/guides/stay-up-to-date/releases/2023-04" />
-<Card title="Update to 2023-03" href="/guides/stay-up-to-date/releases/2023-03" />
```

Comment:
@oredavids should this be removed?

Reply:
Yes, for each new release an older one is removed from this page. The older one will remain available on the [page for all releases](https://docs.gruntwork.io/guides/stay-up-to-date/releases/)